pyvista/plotting/charts.py

```diff
@@ -1364,6 +1364,7 @@ def show(
         window_size=None,
         notebook=None,
         background='w',
+        **kwargs,
```

Comment:
Oof, I am really not a fan of unchecked `**kwargs`, these can really come back to bite us. If they are truly for developer use, then could we make this `developer_kwargs=None` instead?

Reply:
Resolved in 41ae933.